.github/workflows/docker.yml

```diff
@@ -10,12 +10,31 @@ on:
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    env:
+      IAM_ROLE_SESSION_NAME: geodesic-ci
+      AWS_REGION: us-east-1
+      ECR_REGISTRY: public.ecr.aws/h1u4b2i4/
```

Comment:
Is this customizable?

Reply:
Yes but needs to get approved by amazon, put in request a while ago, might need some verification